### DISCUSSION TITLE
Option to autoSuggest queries instead of results?

### DISCUSSION THREAD

**Discussion by rgb-panda** [appbaseio/reactivesearch#1611] [Q&A]
Is there a way to get list of suggestions that are search queries, rather than actual results?

This is supported by search-ui https://github.com/elastic/search-ui/blob/master/ADVANCED.md#example-using-autocomplete-suggestions 

I couldn't find anything within reactiveSearch that allows it. Is there a way to achieve the same result by using a customQuery for example?

Thanks,

**Comment by siddharthlatest** [comment]
@rgb-panda `enablePredictiveSuggestions` prop, which is available for `DataSearch` and `CategorySearch` components can be used to create suggestions based on what the user has typed so far. Instead of displaying the entire matching result it predicts the most likely combination of next words that can be applied based on the search query so far. Tl;dr - The inspiration for this is how Google search's suggestions work.

Read more at: https://docs.appbase.io/docs/reactivesearch/v3/search/datasearch/

> enablePredictiveSuggestions bool [optional] Defaults to false. When set to true, it predicts the next relevant words from a field's value based on the search query typed by the user. When set to false (default), the entire field's value would be displayed. This may not be desirable for long-form fields (where average words per field value is greater than 4 and may not fit in a single line).

**Reply by rgb-panda** [reply]
Thanks @siddharthlatest I remember trying this, but the list of suggestions will map to one document each? or it's a query suggestion that can return multiple documents?
